src/pages/Account/Login.scss

```diff
@@ -2,29 +2,39 @@
 
 .Login {
   &__login-box {
+    width: 250px;
     display: flex;
     flex-flow: column;
     align-items: center;
-    margin-bottom: $spacing--md;
-    padding: $spacing--lg 0;
+    margin: $spacing--md auto;
+    padding: $spacing--lg $spacing--lg;
     box-shadow: inset 0 0 0 3px #6f43ff;
     border-radius: 0.25rem;
+    font-size: $font-size--xl;
+    transition: all 400ms $transition-function;
+    font-weight: $font-weight--500;
+
+    &:hover {
+      background-color: darken($color: $primary, $amount: 20%);
+      cursor: pointer;
+    }
   }
 
+  // @debt removed because rather than the icon being a button, the whole login box is supposed to be a button
```

Comment:
should we remove it instead of commenting out?

Reply:
@0xdevalias asked me to comment it out instead

Reply:
If this was the main `staging`/etc code branch + this was a permanent change then removing would absolutely make sense. But given this is environment specific hacks, I'd rather us comment it out + leave comments with reasoning why so that when we try and diff the environment against the main codebase again, it will be really obvious what was just a hack vs what might be useful to bring back across.